frontend/src/lib/StatusUtils.ts

```diff
@@ -41,6 +42,18 @@ export enum NodePhase {
   OMITTED = 'Omitted',
 }
 
+export const statusProtoMap = new Map<V2beta1RuntimeState, string>([
+  [V2beta1RuntimeState.RUNTIMESTATEUNSPECIFIED, 'Unknown'],
+  [V2beta1RuntimeState.PENDING, 'Pending'],
+  [V2beta1RuntimeState.RUNNING, 'Running'],
+  [V2beta1RuntimeState.SUCCEEDED, 'Succeeded'],
+  [V2beta1RuntimeState.SKIPPED, 'Skipped'],
+  [V2beta1RuntimeState.FAILED, 'Failed'],
+  [V2beta1RuntimeState.CANCELING, 'Canceling'],
+  [V2beta1RuntimeState.CANCELED, 'Canceled'],
+  [V2beta1RuntimeState.PAUSED, 'Paused'],
+]);
```

Comment:
What happen if more runtime states are introduced? How do we track this information and update accordingly?

Reply:
The map is built according to v2 proto. All of the runtime states we currently have is included. If any status not existed in the proto is appeared, we will display `-` instead. That being said, any proto changes will lead to update for this map.